TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/EntityManagerFactory.php

```diff
@@ -97,6 +97,17 @@ public function create()
         $config->setProxyNamespace('TYPO3\Flow\Persistence\Doctrine\Proxies');
         $config->setAutoGenerateProxyClasses(false);
 
+        // Set default host to 127.0.0.1 if there is no other host configured and at least one other necessary option
+        if ($this->settings['backendOptions']['host'] == '') {
+            if (
+                $this->settings['backendOptions']['dbname'] != '' ||
+                $this->settings['backendOptions']['user'] != '' ||
+                $this->settings['backendOptions']['password'] != ''
+            ) {
+                $this->settings['backendOptions']['host'] = '127.0.0.1';
+            }
+        }
+
```

Comment:
I think this change in itself is a good idea but the above code feels too magic. If I don't set a host but everything else the host will be magically 127.0.0.1, I am not really happy with that yet. I would strongly suggest to deprecate this behavior right away and document that the host MUST be set beginning with the next major Flow version.